src/client/auth.js

```diff
@@ -4,7 +4,12 @@ const fs = require('fs').promises
 const mcDefaultFolderPath = require('minecraft-folder-path')
 const path = require('path')
 
-module.exports = async function (client, options) {
+module.exports = function (client, options) {
+  client.username = ''
```

Comment:
Wouldn't it be safer to initialize it to:
```js
    client.username = options.username
```
?

Reply:
I think that would be just as valid, since neither can be trusted to be an actually valid username, but sure we can do this.